scripts/exgfs_atmos_awips_20km_1p0deg.sh

```diff
@@ -51,6 +54,7 @@ while (( icnt < 1000 )); do
    if (( icnt >= 180 )); then
       msg="FATAL ERROR: No GFS pgrb2 file after 30 min of waiting"
       err_exit "${msg}"
+      exit 5
```

Comment:
doesn't `err_exit` cause this to exit already? `exit 5` will never reach if there is an error.

Reply:
It's _supposed_ to, but given `err_exit` doesn't work on some RDHPCS, I put this in (it was on a never-ending loop last night before I fixed something).